src/mumble/Log.cpp

```diff
@@ -102,15 +113,21 @@ LogConfig::LogConfig(Settings &st) : ConfigWidget(st) {
 		twi->setWhatsThis(ColHighlight, tr("Click here to toggle window highlight for %1 events.<br />If checked, "
 										   "Mumble's window will be highlighted for every %1 event, if not active.")
 											.arg(messageName));
+		twi->setWhatsThis(ColMessageThreshold,
+			tr("Click here to toggle thresholding for %1 events. <br /> If checked, notifications for this event type"
+			   "will not be played when above the set number of users.<br />Ensure that the thresholding "
+			   "option below is enabled or this field will not have any effect.")
+				.arg(messageName));
 		twi->setWhatsThis(ColStaticSound, tr("Click here to toggle sound notification for %1 events.<br />If checked, "
 											 "Mumble uses a sound file predefined by you to indicate %1 events. Sound "
 											 "files and Text-To-Speech cannot be used at the same time.")
 											  .arg(messageName));
 		twi->setWhatsThis(ColStaticSoundPath,
-						  tr("Path to sound file used for sound notifications in the case of %1 events.<br />Single "
+						 tr("Path to sound file used for sound notifications in the case of %1 events.<br />Single "
```

Comment:
Unrelated change

Reply:
You guys don't follow the "boy scout rule"? :stuck_out_tongue:

Reply:
It depends. Fixing this is fine, but it should be separated from the main changes. So if you wrap this up in its own `FORMAT` commit, then that'd be fine :)